core/controlplane/config/templates/cloud-config-etcd

```diff
@@ -269,6 +269,7 @@ coreos:
 
             [Service]
             EnvironmentFile=-/etc/etcd-environment
+            EnvironmentFile=-/var/run/coreos/etcdadm/etcd-member.env
```

Comment:
@martinssipenko Hi, thanks for contributing!
Would you mind sharing me your intention to add this line?
Seemed like this env file is read from nowhere.

Reply:
This file is generated by etcdadm

Reply:
Yes! However, I thought we already feed it to etcd somewhere else?
Where is the env file used from when written here?